README.md

```diff
@@ -71,8 +71,7 @@ issue. We thank you in advance for your cooperation.
 
 * C compiler
 * [Go compiler](https://golang.org/dl/)
-* liblzma-dev and libssl-dev packages
-
```

Comment:
Did you remove the extra new line on purpose?

Reply:
No, by mistake. Thanks

Reply:
Fixed now.